### PR TITLE
adding network upgrade information

### PR DESCRIPTION
**Pull request by sbvegan**
**Description**

A clear and concise description of the features you're adding in this pull request.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Review by cpengilly — APPROVED** [review]

File: pages/builders/node-operators/network-upgrades/delta.mdx [hunk 1]
```diff
@@ -13,19 +13,33 @@ The following information is how you can prepare your nodes for the Delta upgrad
...
+  The Delta mainnet network upgrade for OP Mainnet will activate at **1708560000 Thu Feb 22 00:00:00 UTC 2024**.
 </Callout>
 
-## Testnet Information 
+### Required Action
 
-<Callout type="warning">
-    *   The Delta testnet network upgrade for OP Goerli will activate at **1703116800 Thu Dec 21 00:00:00 UTC 2023**.
-    *   The Delta testnet network upgrade for OP Sepolia will activate at **1703203200 Fri Dec 22 00:00:00 UTC 2023**.
+You must upgrade your nodes to the following versions prior to
```
Comment (trianglesphere) [inline pages/builders/node-operators/network-upgrades/delta.mdx:21]:
```suggestion
You must upgrade your nodes to the following versions prior to Thu Feb 22 00:00:00 UTC 2024
```

File: pages/builders/node-operators/network-upgrades/delta.mdx [hunk 2]
```diff
@@ -13,19 +13,33 @@ The following information is how you can prepare your nodes for the Delta upgrad
...
+*   Check that Delta time is set to `1708560000` in the op-node logs
+
+## Testnet Information
+
+<Callout type="info">
+  *   The Delta testnet network upgrade for OP Goerli went live at **Thu, 2023-12-21 at 00:00:00 UTC**.
+  *   The Delta testnet network upgrade for OP Sepolia went live at **Fri, 2023-12-22 at 00:00:00 UTC**.
 </Callout>
 
 ### Required Action
```
Comment (trianglesphere) [inline pages/builders/node-operators/network-upgrades/delta.mdx:40]: Do we want to remove this a section or add in the timestamp?

File: pages/builders/node-operators/network-upgrades/ecotone.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+*   Upgrade your `op-node` to version: [v1.5.0](https://github.com/ethereum-optimism/optimism/releases/tag/v1.5.0)
+*   Upgrade your `op-geth` to version: [v1.101305.3](https://github.com/ethereum-optimism/op-geth/releases/tag/v1.101305.3)
+
+### Verifying Your Configuration on OP Goerli
+
+You can verify your node is configured correctly with the following:
+
+*   `op-node` will log its configuration at startup
+*   Check that Ecotone time is set to `1707238800` in the op-node logs
```
Comment (trianglesphere) [inline pages/builders/node-operators/network-upgrades/ecotone.mdx:41]: op-geth will also log Ecotone & Cancun at the same timestmap.

Comment (sbvegan) [reply pages/builders/node-operators/network-upgrades/ecotone.mdx:41]: Would it be the same for op-node too?

File: pages/builders/node-operators/network-upgrades/delta.mdx (hunk 2 above)
Comment (sbvegan) [reply pages/builders/node-operators/network-upgrades/delta.mdx:40]: I think I'm just going to remove callouts for activated network upgrades. One less thing to do and we have it on the overview page anyways.

File: pages/builders/node-operators/network-upgrades/delta.mdx (hunk 1 above)
Comment (sbvegan) [reply pages/builders/node-operators/network-upgrades/delta.mdx:21]: oops

**Pull request merged**